cameo/core/reaction.py

```diff
@@ -187,6 +188,13 @@ def __deepcopy__(self, memo):
     def lower_bound(self):
         return self._lower_bound
 
+    @property
+    def functional(self):
```

Comment:
add a docstring here, although the name is quite descriptive a definition would help, e.g, a gene is functional unless the gpr states otherwise